_schemaV2.graphql

```diff
@@ -11054,11 +11076,25 @@ enum HomeViewComponentBackgroundImageURLVersion {
   WIDE
 }
 
+type HomeViewComponentBehaviors {
+  # Represents the behavior of the view all button
+  viewAll: HomeViewComponentBehaviorsViewAll
+}
+
+type HomeViewComponentBehaviorsViewAll {
+  # Text for the CTA of the view all button
+  buttonText: String!
+
+  # href of the view all button
+  href: String!
+}
```

Comment:
**questions (non-blocking):** Nice 👏🏽 

Some further questions:

 - Is this meant to handle both of the current (or future) kinds of placements for the view-all UI? Or would we distinguish between them in the response?
    <img width=200 src="https://github.com/user-attachments/assets/aa85c7ee-ff2e-4b4b-87d7-f639745ef273" />


 - This clearly sets out the behavior where we want to go to a predefined route. But what about the other case: where we don't have a route but instead just expand to a full-screen view of the data. Would that be indicated by the absence of `href`? Or should it be signaled by the presence of some other attribute, with some more explicit gql modeling?

(No need to answer in this PR — and a reminder that we can introduce breaking changes till Milestone 2)